src/FilterButtonList.js

```diff
@@ -73,50 +72,48 @@ let GroupBox = ({ nodeJoinColor, children, nested, className }) => (
   </Flex>
 )
 
-let FilterButtonList = InjectTreeNode(
-  observer(
```

Comment:
this whole diff is just the indent change from removing `observer` (plus renaming `injectTreeNode`)